cloud/src/autogluon/cloud/predictor/predictor.py

```diff
@@ -238,12 +244,40 @@ def _upload_fit_artifact(
             bucket=cloud_bucket,
             key_prefix=util_key_prefix
         )
-        return train_input, tune_input, config_input
+
+        images_input = images
+        if images is not None:
+            if is_s3_url(images):
+                fileexts = ('.tar.gz', '.bz2', '.zip')  # More extensions?
+                assert images.endswith(fileexts), 'Please provide a s3 path to a compressed file'
+            else:
+                image_zip_filename = images
+                if os.path.isdir(images):
+                    image_zip_filename = os.path.basename(os.path.normpath(images))
+                    zipfolder(image_zip_filename, images)
+                    image_zip_filename += '.zip'
+                else:
+                    assert is_compressed_file(images), 'Please provide a compressed file or a folder containing the images'
+                logger.log(20, 'Uploading images ...')
+                images_input = self.sagemaker_session.upload_data(
+                    path=image_zip_filename,
+                    bucket=cloud_bucket,
+                    key_prefix=util_key_prefix
+                )
+                logger.log(20, 'Images uploaded successfully')
+        inputs = dict(train=train_input, config=config_input)
+        if tune_input is not None:
+            inputs['tune'] = tune_input
+        if images_input is not None:
+            inputs['images'] = images_input
+
+        return inputs
 
     def fit(
         self,
         predictor_init_args,
         predictor_fit_args,
+        image_path=None,
```

Comment:
Just want to ask if `image_path` is a must to provide. If it is a must, do we want to set a default value here?

Reply:
In the `ImagePredictor.fit()`, `image_path` is actually required. Here is the `fit()` for general `CloudPredictor`.
You can also imagine when we implement multimodality, this `image_path` could be optional